pkgs/top-level/python-packages.nix

```diff
@@ -5111,6 +5111,10 @@ in {
 
   importlib-resources = callPackage ../development/python-modules/importlib-resources {};
 
+  easywatch = callPackage ../development/python-modules/easywatch { };
```

Comment:
These should sorta be alphabetically sorted.

Reply:
I'll try to find the correct place then

( This is probably the least sorted list in all nixpkgs :-P )